src/Components/Header.tsx

```diff
@@ -63,25 +102,51 @@ export const HeaderComponent = ({
       </VisuallyHidden>
       <Header>
         <HeaderContents>
-          {homeButton && (
-            <Link href={`/`} passHref>
-              <ButtonLink>
-                <Image
-                  src="/img/home-icon.png"
-                  alt="Home"
-                  height={44}
-                  width={44}
-                />
-              </ButtonLink>
-            </Link>
-          )}
           <Title>{title}</Title>
           {filterButton && (
             <Link href={`/filter`} passHref>
               <ButtonLinkText>Filter</ButtonLinkText>
             </Link>
           )}
         </HeaderContents>
+        <Nav>
+          <LinkList>
+            <li>
+              <Link href={`/`} passHref>
+                <NavPill activePage={currentPage === '/' || currentPage === ''}>
+                  All
+                </NavPill>
+              </Link>
+            </li>
+            <li>
+              <Link href={quizComplete ? '/quiz/results' : '/quiz'} passHref>
+                <NavPill activePage={currentPage.includes('quiz')}>
+                  Quiz
+                </NavPill>
+              </Link>
+            </li>
+            <li>
+              <Link href={`/saved`} passHref>
+                <NavPill activePage={currentPage === '/saved'}>
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      strokeWidth="2"
+                      d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z"
+                    />
+                  </svg>
```

Comment:
It'd be nice to have this star as a reusable component, but I think that's carded so it's ok to leave as this for now.

Reply:
Yep, hoping to defer to https://trello.com/c/XzYTtjUg/102-standardise-the-star-icon-used-across-the-app-v-e so we don't have to split it if the save symbol becomes different to to the rating symbol. It shouldn't be hard to round up where it's used.